compiler/AST/expr.cpp

```diff
@@ -6232,6 +6278,23 @@ GenRet CallExpr::codegenBasicPrimitiveExpr() const {
   return codegenCallExpr(primitive->name, args);
 }
 
+bool CallExpr::isExternStarTuple(Symbol* formal, Expr* actual) const {
```

Comment:
I'm a little surprised that this method isn't called isRefExternStarTuple or something along those lines, but I see it does the same as before. Is there some reason you chose not to include 'Ref' in the name of this method? (e.g. that you're going to remove the ref-check in a subsequent PR, or on the principle that we're handling ref wrong or...?)